src/XMakeTasks/Exec.cs

```diff
@@ -58,13 +58,25 @@ public Exec()
         private readonly List<ITaskItem> _nonEmptyOutput = new List<ITaskItem>();
         private Encoding _standardErrorEncoding;
         private Encoding _standardOutputEncoding;
+        private string _command;
 
         #endregion
 
         #region Properties
 
         [Required]
-        public string Command { get; set; }
+        public string Command
+        {
+            get { return _command; }
+            set
+            {
+                _command = value;
+                if (Path.DirectorySeparatorChar == '/')
```

Comment:
I thought I would go with the existing convention being used in the [file](https://github.com/microsoft/msbuild/blob/xplat/src/XMakeTasks/Exec.cs#L196), but I guess `IsUnixLike` would be better. And looks like it is being used in the [file](https://github.com/microsoft/msbuild/blob/xplat/src/XMakeTasks/Exec.cs#L553) already.